### PR TITLE
Mesh constructor texture fix

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
This is fix for problem that was discovered in v5: #4721 . it can be hardly reproduced at https://jsfiddle.net/Hackerham/cpL5n1u9/13/ . 

I'm trying to get the correct order of operations and throttling to show the bug

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
I don’t see what’s wrong with the fiddle. What happens when the big shows up?

File: src/mesh/Plane.js [hunk 1]
```diff
@@ -108,13 +108,14 @@ export default class Plane extends Mesh
...
-        // wait for the Plane ctor to finish before calling refresh
-        if (this._ready)
+        // constructor texture update - stop 1
+        if (!this._ready)
         {
-            this.refresh();
+            return;
         }
+
+        Mesh.prototype._onTextureUpdate.call(this);
```
Comment (bigtimebuddy) [inline src/mesh/Plane.js]: Why not `super._onTextureUpdate()`?

**Comment by ivanpopelyshev** [comment]
That's better fix, if its ok im gonna put it in v5 too.

**Comment by ivanpopelyshev** [comment]
The fix isn't necessary, it looks like `TextureMatrix` tracks initial load as good as `onTextureUpdate`

**Pull request closed without merging**